docs/source/install/config/config.rst

```diff
@@ -98,6 +98,19 @@ In :github_st2:`/etc/st2/st2.conf <conf/st2.prod.conf>` include the following se
 
 The ``#RMQ_VHOST`` property is optional and can be left blank.
 
+|st2| also supports SSL/TLS to encrypt RabbitMQ connections. To enable SSL, you need to pass
+``?ssl=true`` query parameter at the end of the connection URL.
+
+For example:
+
+.. code-block:: ini
+
+   url = amqp://guest:guest@127.0.0.1:5671/?ssl=true
```

Comment:
Is there anything else that's required on ST2 conf side to configure certificates/trusted/chain to communicate with SSL-enabled RabbitMQ server, the right way?

Just for reference, I'm looking at RabbitMQ clients from the examples like:
https://www.rabbitmq.com/ssl.html#java-client-connecting-with-peer-verification
https://pika.readthedocs.io/en/latest/examples/tls_mutual_authentication.html
http://docs.celeryproject.org/projects/kombu/en/latest/reference/kombu.connection.html

Reply:
This change only enables secure communication to the server using TLS, but it doesn't do peer verification and it also doesn't support client certificate based authentication.

For that we will need to modify the code (similar to the user contribution for MongoDB not too long ago).

The plan is to use the same configuration option names as we use for MongoDB in the ``database`` section in st2.conf (for consistency reasons).

But yeah, that will be a separate change. I will open an issue in StackStorm/st2 to track it.

Reply:
Would be good to add a small note about that part in st2docs then.